Formula/talisman.rb

```diff
@@ -0,0 +1,30 @@
+class Talisman < Formula
+  desc "Detects secrets checked into git repository before commit or push"
+  homepage "https://thoughtworks.github.io/talisman/"
+  url "https://github.com/thoughtworks/talisman/archive/v0.3.5.tar.gz"
+  sha256 "7c71d5e4a98a5061598fadc9155ee02ae27d0a85f912273360d987a65d2c7362"
+
+  depends_on "go" => :build
+  depends_on "gox" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "on"
+    talisman_path = buildpath/"src/github.com/thoughtworks/talisman"
+    talisman_path.install buildpath.children
+
+    cd talisman_path do
+      arch = MacOS.prefer_64_bit? ? "amd64" : "386"
```

Comment:
https://brew.sh/2019/01/09/homebrew-1.9.0/ with linux joining the brew mix, do we really need all this specific macOS stuff?

Reply:
+ we dropped non 64 bit support in https://github.com/Homebrew/homebrew-core/pull/35707